doc/getstarted/basic_usage/index_en.rst

```diff
@@ -96,6 +93,6 @@ In PaddlePaddle, training is just to get a collection of model parameters, which
     .. image:: parameters.png
         :align: center
 
-Although starts from a random guess, you can see that value of ``w`` changes quickly towards 2 and ``b`` changes quickly towards 0.3. In the end, the predicted line is almost identical with real answer.
+Although starts from a random guess, you can see that value of :math:`w` changes quickly towards 2 and :math:`b` changes quickly towards 0.3. In the end, the predicted line is almost identical with the real answer.
 
-There, you have recovered the underlying pattern between ``X`` and ``Y`` only from observed data.
+There, you have recovered the underlying pattern between :math:`X` and :math:`Y` only from observed data.
```

Comment:
Finally, we have learned the underlying mapping function from :math:`X` to :math:`Y` from observed data.